common/app/common/commercial/ContainerModel.scala

```diff
@@ -23,7 +30,7 @@ case class ContainerContent(
 
 object ContainerModel {
 
-  def fromPressedCollection(collection: PressedCollection): ContainerModel = {
+  def fromPressedCollection(edition: Edition)(collection: PressedCollection): ContainerModel = {
```

Comment:
It's so that I can turn it into a function that can be passed into a map in frontBody.scala.html line 40.